docker/yum-autodetect/Dockerfile

```diff
@@ -2,7 +2,10 @@ FROM centos:7
 
 WORKDIR /ahab-docker
 
-RUN yum -y install epel-release which python3-pip
+RUN yum -y install epel-release python3-pip
+
+#Required for autodetect to work
+RUN yum -y install which
```

Comment:
Why not install that as a part of line 5?

Reply:
Yup I could and that is how it was ... Thought maybe it'd be nice to be explicit since that image doesn't have which by default

Reply:
Makes sense. My only thought was in how Docker caches layers, but it doesn't really matter!